src/auth/googleauth.ts

```diff
@@ -59,6 +64,16 @@ export class GoogleAuth {
   set cachedProjectId(projectId: string) {
     this._cachedProjectId = projectId;
   }
+  // To save the contents of the JSON credential file
+  private _jsonContent: JWTInput;
+
+  get jsonContent(): JWTInput {
```

Comment:
These accessors are simple pass throughs to `_jsonContent`. Just make the property public. See style guide section #getters-and-setters-accessors.

Reply:
Made public and removed getter and setter